### PR TITLE
Feature/esp32 adaptation

### PR DESCRIPTION
**Pull request by dchilot**

### CODE REVIEW

File: arduino/main/main.ino [hunk 1]
```diff
@@ -23,185 +30,172 @@ bool OLD_KEY1_PRESENT = false;
...
 uint8_t const VOLTAGE_RATIO = 11;
 
-// I2C
-uint8_t const I2C_ADDRESS = 1;
+// BUZZER
+uint8_t const BUZZ_PIN = 13;
+
+double const ADC_MAX_VALUE = 4095.0;
+
+class FakeTagFinder
```
Comment (dchilot) [inline arduino/main/main.ino]: This probably needs to go in its own file(s).

File: arduino/main/main.ino [hunk 2]
```diff
@@ -23,185 +30,172 @@ bool OLD_KEY1_PRESENT = false;
...
 harpi::Motor MOTOR1(MOTOR1_PWM_PIN, MOTOR1_IN1_PIN, MOTOR1_IN2_PIN);
-harpi::Motor MOTOR2(MOTOR2_PWM_PIN, MOTOR2_IN1_PIN, MOTOR2_IN2_PIN);
+harpi::Motor MOTOR2(MOTOR2_PWM_PIN, MOTOR2_IN1_PIN, MOTOR2_IN2_PIN, harpi::LogicToMotion::Pin1LowIsForward);
 harpi::UltraSound US(US_TRIG_PIN, US_ECHO_PIN);
 harpi::TagFinder TAG_FINDER(RFID_SDA_PIN, RFID_RST_PIN);
 Adafruit_NeoPixel LEDS = Adafruit_NeoPixel(LED_COUNT, LED_PIN, NEO_GRB + NEO_KHZ800);
 harpi::Led LED1(0, LEDS);
 harpi::Led LED2(1, LEDS);
 
+int channel = 0;
```
Comment (dchilot) [inline arduino/main/main.ino]: Sound could be in a dedicated class.

File: arduino/main/main.ino [hunk 3]
```diff
@@ -23,185 +30,172 @@ bool OLD_KEY1_PRESENT = false;
...
+  {
+    ledcWriteNote(channel, note, 4);
+    delay(400);
+  }
+  ledcWriteNote(channel, NOTE_C, 5);
+  delay(400);
+  silence();
 }
 
 double DISTANCE = 0;
```
Comment (dchilot) [inline arduino/main/main.ino:216]: This should probably be a local variable in loop.

File: arduino/main/main.ino [hunk 4]
```diff
@@ -23,185 +30,172 @@ bool OLD_KEY1_PRESENT = false;
...
 double VOLTAGE = 0;
```
Comment (dchilot) [inline arduino/main/main.ino:217]: This should probably be a local variable in loop.

File: arduino/main/main.ino [hunk 5]
```diff
@@ -23,185 +30,172 @@ bool OLD_KEY1_PRESENT = false;
...
 uint8_t RFID_CARDS_NUMBERS = 0;
```
Comment (dchilot) [inline arduino/main/main.ino:218]: This does not look to be used.

File: arduino/main/main.ino (hunk 1 above)
Comment (miludmann) [reply arduino/main/main.ino]: Done

**Pull request merged**